sku.0/sys.server/compiled/game/script/library/jedi.java

```diff
@@ -2214,55 +2214,14 @@ public static void initializeCrystal(obj_id crystal, int level) throws Interrupt
     public static boolean isForceSensitive(obj_id player) throws InterruptedException
     {
         String classTemplate = getSkillTemplate(player);
-        if (classTemplate != null && classTemplate.startsWith("force"))
-        {
-            return true;
-        }
-        return false;
+        return (classTemplate != null && classTemplate.startsWith("force")) ? true : false;
     }
     public static boolean isForceSensitiveLevelRequired(obj_id player, int requiredLevel) throws InterruptedException
     {
-        int playerLevel = getLevel(player);
-        if (isForceSensitive(player) && playerLevel >= requiredLevel)
-        {
-            return true;
-        }
-        return false;
+        return (isForceSensitive(player) && getLevel(player) >= requiredLevel) ? true : false;
     }
     public static boolean hasAnyUltraCloak(obj_id player) throws InterruptedException
     {
-        if (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_UP))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_UP))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN))
-        {
-            return true;
-        }
-        return false;
+        return (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_UP) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_UP) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN)) ? true : false;

```

Comment:
This is "shorter", but causes difficulty when trying to read it.  I would leave it as is or at least make the change multiple lines broken by the logical OR's.

Reply:
I guess I just didn't see it as difficult to read.  I just look at it as a reduction in lines of code.  Seemed worth doing, but thats me.